<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_interface</class>
 <widget class="QMainWindow" name="user_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>791</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#QWidget {
background:#404040;
}
QTabWidget::pane { /* The tab widget frame */
border-top: 5px solid #535353;
}
QTabWidget::tab-bar {
left: 0px; /* move to the right by 5px */
}
/* Style the tab using the tab sub-control. Note that it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
background: #0000ff;
border: 2px solid #535353;
border-bottom-color: #535353; /* same as the pane color */
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 120px;
padding: 8px;
color: #fff;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:selected, QTabBar::tab:hover {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #00aaff, stop: 0.4 #00aaff, stop: 0.5 #00aaff, stop: 1.0 #00aaff);
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #535353; /* same as pane color */
color: #dcdcdc;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}
QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #ccc;
  border-radius: 4px;
  box-sizing: border-box;
}</string>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QLineEdit" name="id_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Pr√©nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Salaire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>MDP</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="mail_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>330</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="mdp_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>400</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>452</x>
         <y>180</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #fff;
background:#0000ff;
border-radius: 15px;
font-size:40px;
font-family:Arial;
}
QPushButton:hover {
color:#fff;
border:15px;
background:#00aaff;
}
</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="salaire_line">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>260</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>711</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>321</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #fff;
background:#0000ff;
border-radius: 15px;
font-size:40px;
font-family:Arial;
}
QPushButton:hover {
color:#fff;
border:15px;
background:#00aaff;
}</string>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
  width: 100%;
  padding: 10px 15px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #dcdcdc;
  border-radius: 4px;
  box-sizing: border-box;
  background: #0000ff;
  color: #dcdcdc;
  font-size: 20px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <item>
        <property name="text">
         <string>nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prenom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>salaire</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Rechercher</string>
      </attribute>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>621</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>20</y>
         <width>251</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #fff;
background:#0000ff;
border-radius: 15px;
font-size:40px;
font-family:Arial;
}
QPushButton:hover {
color:#fff;
border:15px;
background:#00aaff;
}
</string>
       </property>
       <property name="text">
        <string>rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>201</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>301</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Saisir l'id de l'employer √† rechercher      :</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Supprimer</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>411</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Saisir l'ID de l'employ√© √† suprimer  :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>211</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>230</y>
         <width>211</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(0, 170, 255);
color: #fff;
background:#0000ff;
border-radius: 15px;
font-size:40px;
font-family:Arial, Helvetica, sans-serif;
}
QPushButton:hover {
color:#fff;
border:15px;
background:#00aaff;
}
</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Modifier</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>-20</y>
         <width>481</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>Saisir l'ID de l'employ√© √† modifier   :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>20</y>
         <width>211</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>180</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #fff;
background:#0000ff;
border-radius: 15px;
font-size:40px;
font-family:Arial;
}
QPushButton:hover {
color:#fff;
border:15px;
background:#00aaff;
}
</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>150</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>nouveau nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>nouveau prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>270</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>nouveau salaire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>nouvelle adresse mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>380</y>
         <width>201</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  color: #616161;
  font-size: 15px;
font-family:Arial, Helvetica, sans-serif;
}</string>
       </property>
       <property name="text">
        <string>nouveau mot de passe</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="new_nom">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="new_prenom">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>200</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="new_salaire">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>260</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="new_mail">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>320</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="new_mdp">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>380</y>
         <width>141</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
